src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -100,6 +100,7 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                 Location interfaceLocation = interfaceData.Syntax.GetLocation();
                 var methods = ImmutableArray.CreateBuilder<(MethodDeclarationSyntax Syntax, IMethodSymbol Symbol, int Index, Diagnostic? Diagnostic)>();
                 int methodVtableOffset = interfaceContext.MethodStartIndex;
+                var i = methodVtableOffset;
```

Comment:
Can we just use methodVTableOffset for this instead of introducing another variable? I think the two variables represent the same concept here.